src/main/java/org/interledger/codecs/AbstractCodec.java

```diff
@@ -19,11 +19,12 @@ public AbstractCodec(final int typeId) {
   }
 
   /**
-   * Read an object from the buffer according to the rules defined in the {@link CodecContext}.
+   * Read an object of the buffer according to the rules defined in the {@link CodecContext}.
```

Comment:
eh, we went from "Read an object from the buffer..." to "Read an object of the buffer..."? or am i not reading the diff correctly?